tests/features/unittest_aliases_test.py

```diff
@@ -7,14 +7,21 @@
 
 
 @pytest.mark.parametrize(
-    ('s',),
+    's',
     (
         pytest.param(
             'class ExampleTests:\n'
             '    def test_something(self):\n'
             '        self.assertEqual(1, 1)\n',
             id='not a deprecated alias',
         ),
+        # https://github.com/asottile/pyupgrade/issues/749
+        'unittest.findTestCases(Tests, "arg")',
+        'unittest.findTestCases(Tests, prefix="arg")',
+        'unittest.makeSuite(Tests, "arg")',
+        'unittest.makeSuite(Tests, prefix="arg")',
+        'unittest.getTestCaseNames(Tests, "arg")',
+        'unittest.getTestCaseNames(Tests, prefix="arg")',
```

Comment:
don't need a test for each of them since there's no special logic between them.  just the minimal to demonstrate the change